TGC/events/BanksAndUniversities.txt

```diff
@@ -0,0 +1,406 @@
+        modifier = { factor = 0.6
+                     owner = { money = 200000 } }
+        modifier = { factor = 0.5 
+					 owner = { money = 300000 } }
+
+        modifier = { factor = 4                                         #Lower chance if investing low on administration
+                    NOT = { administration_spending = 0.3 } }
+        modifier = { factor = 8
+                    NOT = { administration_spending = 0.15 } }
+
+		modifier = { factor = 0.2										#Higher chance if good RGO
+					OR = {	produces = precious_goods
+					 		produces = precious_metal } }
+		modifier = { factor = 0.6
+					OR = {	produces = iron
+							produces = coal
+							produces = oil
+					 		produces = rubber } }
+		modifier = { factor = 0.8
+					OR = {	produces = sulphur
+					 		produces = cotton } }
+
+        modifier = { factor = 3                                         #Lower chance if the life rating is low
+                     NOT =  { life_rating = 10 } }
+        modifier = { factor = 2   
+                     NOT =  { life_rating = 15 } }
+        modifier = { factor = 1.5   						
+                     NOT =  { life_rating = 20 } }
+
+        modifier = { factor = 0.8                                       #Higher chance if province has other economic buildings
+                    has_building = infrastructure }
+        modifier = { factor = 0.8                                      
+                    has_building = university }
+        
+        modifier = { factor = 0.7                                       #Increase chance if the AI is reasearching Bank tech        
+                    stock_exchange = 1 }                                
+        modifier = { factor = 0.7                                       
+                    business_banks = 1 }                            
+        modifier = { factor = 0.7                                       
+                    investment_banks = 1 }                              
+        modifier = { factor = 0.7                                       
+                    bank_inspection_board = 1 }
+        modifier = { factor = 0.7                                       
+                    mutual_funds = 1 }     
+    }
+
+	option = {
+		name = "AIEVENT" 
+        owner = { treasury = -3000 }
+		bank = 1
+
+		random_pop = {                     #Paying workers
+			limit = { strata = poor	}							
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = middle }
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = rich }
+			money = 500
+		}
+	}
+}
+
+## Abusive Banking Practices
+province_event = {
+	id = 6202
+	title = "EVTNAME6202"
+	desc = "EVTDESC6202"
+
+	trigger = {
+		has_building = bank
+		NOT = { administration_spending = 0.5 }
```

Comment:
are you sure this works as a province-level trigger? I think it's a country scope in which case you gotta check this for the owner of the province rather than the province itself

Reply:
I was discussion with slav earlier, for some reason administration and education spending will work normally in a province event. I realized it shouldnt but it was working anyway later in development, and idk if I should ad the owner tag or not given the weird situation